### PR TITLE
Latest qa

### LINKED ISSUE #1629: Latest QA
Make sure WComponents is using:

- latest qa-parent
- travis-ci
- sonarcloud
- correct badges

### PR DESCRIPTION
**Pull request by jonathanaustin**
Closes #1629 

- Latest qa-parent
- Move to travis-ci.com
- Integrate sonarcloud & codacy
- New badges
- Fix some QA issues

Note - Theme OptionalTests not running on Travis as Saucelabs and Intern are not working at the moment

### CODE REVIEW

**Comment by jonathanaustin** [comment]
The sonarcloud check fail looks bad but is caused by low code coverage. As this is is the first PR using sonarcloud I thought it would be OK with letting this through.

**Comment by jonathanaustin** [comment]
Created a quality gate on sonarcloud that allows a minimum coverage of 70%.

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+ - mvn install -DskipTests=true -Dmaven.javadoc.skip=true -B -V -DskipOptionalQA=true -Dwc.pmd.verbose=false
+
+## Activate skipThemeOptionalTests until Intern and Saucelabs integration fixed
 script:
- - mvn test
+ - mvn package sonar:sonar -Dsonar.projectKey="bordertech-wcomponents" -PskipThemeOptionalTests -PskipCoreOptionalTests
+
+## Send Coverage to Codacy
+after_success:
+  - java -jar codacy-coverage-reporter-4.0.5-assembly.jar report -l Java -r wcomponents-core/target/site/jacoco/jacoco.xml
```
Comment (ricksbrown) [inline .travis.yml]: Style nit: here and elsewhere in this PR no newline at end of file.
I don't know how much we care but github cares and github is my friend.

Comment (jonathanaustin) [reply .travis.yml]: Happy to fix.

**Pull request merged**